routemaster/controllers/subscriber.rb

```diff
@@ -23,8 +23,7 @@ class Subscriber < Sinatra::Base
         end
 
         topics = data['topics'].map do |name|
-          Models::Topic.find(name) ||
-          Models::Topic.new(name: name, publisher: nil)
+          Models::Topic.find_or_create!(name: name)
```

Comment:
Sorry I don't see where the exception is captured, the `rescue` statement is executed after this has already raised, and since it's explicit in a begin block it will miss it. 

> Well, why not? Hopefully it helped readability for you? Not sure I follow :)

I suppose it did, but upon first inspection I thought _are we including active_record/relation somewhere, is this expecting rails to be part of the project?_; As I said just a minor thing, I don't have any better suggestion so... 👍